clang/lib/CIR/CodeGen/CIRGenerator.cpp

```diff
@@ -31,9 +34,14 @@ void CIRGenerator::Initialize(ASTContext &astCtx) {
 
   this->astCtx = &astCtx;
 
-  cgm = std::make_unique<CIRGenModule>(*mlirCtx, astCtx, codeGenOpts, diags);
+  mlirCtx = std::make_unique<mlir::MLIRContext>();
+  mlirCtx->getOrLoadDialect<mlir::cir::CIRDialect>();
```

Comment:
I'm surprised to see something that's named `get()` and doesn't actually look at the return value or accept any arguments; what's going on here?

Reply:
There's a `loadDialect` equivalent that we can switch to.

Reply:
Fixed.  Changed `getOrLoadDialect` to `loadDialect`.